tests/models/hydrology/test_above_ground.py

```diff
@@ -215,3 +215,18 @@ def test_estimate_interception():
     exp_result = np.array([0.0, 1.180619, 5.339031])
 
     np.testing.assert_allclose(result, exp_result)
+
+
+def test_distribute_monthly_rainfall():
+    """Test that randomly generated numbers are reproducible."""
+    from virtual_rainforest.models.hydrology.above_ground import (
+        distribute_monthly_rainfall,
+    )
+
+    monthly_rain = np.array([0.0, 20.0, 200.0])
```

Comment:
If you moved the seed here then I think the following would test what you are currently testing:
```python
np.random.seed(42)
result = distribute_monthly_rainfall(monthly_rain, 10)
np.random.seed(42)
result1 = distribute_monthly_rainfall(monthly_rain, 10)
```